Formula/got.rb

```diff
@@ -0,0 +1,22 @@
+class Got < Formula
+  desc "Version control system"
+  homepage "https://gameoftrees.org"
+  url "https://gameoftrees.org/releases/portable/got-portable-0.75.tar.gz"
+  sha256 "b35324893c9aefbd6bf49602a6c9bb9d6e20670aff3c9734fefead0205bb9cb2"
+  license "ISC"
+
+  depends_on "ncurses" => :build
+  depends_on "openssl@3" => :build
+  depends_on "ossp-uuid" => :build
+
+  uses_from_macos "bison" => :build
+
+  def install
+    system "./configure", *std_configure_args, "--disable-silent-rules"
+    system "make", "install"
+  end
+
+  test do
+    system "#{bin}/got", "--version"
```

Comment:
Can you propose a more substantial test than checking the version or calling for help? These tests rarely ever fail even for broken installations.

>We want tests that don’t require any user input and test the basic functionality of the application. For example `foo build-foo input.foo` is a good test and (despite their widespread use) `foo --version` and `foo --help` are bad tests. 

Ref. https://docs.brew.sh/Formula-Cookbook#add-a-test-to-the-formula

Reply:
I borrowed the test implementation from https://github.com/Homebrew/homebrew-core/pull/106492.